dashboard/app/models/user.rb

```diff
@@ -889,6 +892,11 @@ def User.track_level_progress_sync(user_id, level_id, script_id, new_result, sub
       user_level.save!
     end
 
+    # Create peer reviews after submitting a peer_reviewable solution
+    if user_level && user_level.submitted && Level.cache_find(level_id).try(:peer_reviewable)
+      PeerReview.create_for_submission(user_level, level_source_id)
+    end
```

Comment:
This could move to `:after_save` in `UserLevel` if I start populating `UserLevel#level_source_id`.